grakn-dist/src/grakn

```diff
@@ -0,0 +1,662 @@
+  grakn=$?
+  # log
+  if [ $grakn -ne 0 ] ; then
+    echo "Unable to start Grakn"
+  fi
+  return $grakn
+}
+
+grakn_server_start_queue() {
+  queue_start_and_wait_until_ready
+  queue=$?
+  # log
+  if [ $queue -ne 0 ] ; then
+    echo "Unable to start Queue"
+  fi
+  return $queue
+}
+
+grakn_server_start_storage() {
+  storage_start_and_wait_until_ready
+  storage=$?
+  # log
+  if [ $storage -ne 0 ] ; then
+    echo "Unable to start Storage"
+  fi
+  return $storage
+}
+
+grakn_server_start() {
+  grakn_server_start_storage
+  local storage=$?
+
+  grakn_server_start_queue
+  local queue=$?
+
+  grakn_server_start_grakn
+  local grakn=$?
+
+  local status=
+  if [ $storage -ne 0 ] || [ $queue -ne 0 ] || [ $grakn -ne 0 ]; then
+    status=1
+  else
+    status=0
+  fi
+
+  return $status
+}
+
+grakn_server_help() {
+  echo "Usage: grakn server COMMAND
+
+COMMAND:
+start [grakn|queue|storage]  Start Grakn (or optionally, only one of the component)
+stop [grakn|queue|storage]   Stop Grakn (or optionally, only one of the component)
+status                         Check if Grakn is running
+clean                          DANGEROUS: wipe data completely
+
+Tips:
+- Start Grakn with 'grakn server start'
+- Start or stop only one component with, e.g. 'grakn server start storage' or 'grakn server stop storage', respectively
+"
+}
+grakn_server_status() {
+  storage_is_storage_running
+  if [ $? -eq 0 ] ; then
+    echo "Casssandra: RUNNING"
+  else
+    echo "Casssandra: NOT RUNNING"
+  fi
+
+  queue_is_queue_running
+  if [ $? -eq 0 ] ; then
+    echo "Queue: RUNNING"
+  else
+    echo "Queue: NOT RUNNING"
+  fi
+
+  grakn_is_grakn_ready
+  if [ $? -eq 0 ] ; then
+    echo "Grakn: RUNNING"
+  else
+    echo "Grakn: NOT RUNNING"
+  fi
+}
+
+# TODO
+# better to remove this functionality in production.
+# Having a very accessible full deletion functionality like this is dangerous
+grakn_server_clean() {
+  storage_is_storage_running
+  local is_storage_running=$?
+
+  queue_is_queue_running
+  local is_queue_running=$?
+
+  grakn_is_grakn_ready
+  local is_grakn_running=$?
+
+  if [ $is_storage_running -eq 0 ] || [ $is_queue_running -eq 0 ] || [ $is_grakn_running -eq 0 ] ; then
+    echo "Grakn is still running! Please do a shutdown with 'grakn server stop' before performing a cleanup."
+  else
+    echo -n "Are you sure you want to delete all stored data and logs? [y/N] " >&2
+    read response
+    if [ "$response" != "y" -a "$response" != "Y" ]; then
+        echo "Response \"$response\" did not equal \"y\" or \"Y\".  Canceling clean operation." >&2
+        return 0
+    fi
+
+    echo -n "Cleaning Storage..."
+    storage_clean_storage
+    echo "SUCCESS"
+
+    queue_start_and_wait_until_ready
+    queue_clean_queue
+    grakn_server_stop_queue
+    echo "Cleaning Queue...SUCCESS"
+
+    echo -n "Cleaning Grakn..."
+    grakn_clean_grakn
+    echo "SUCCESS"
+  fi
+}
+
+grakn_version() {
+  echo "${GRAKN_VERSION}"
```

Comment:
This should query Grakn for the internal version.

Reply:
fixed on 69df8037287025673484bb0b6a1a1ff3af1b0417

Reply:
I also add a command `./graql version`, which prints Grakn version. we didn't have this before